partner_compassion/models/partner_compassion.py

```diff
@@ -318,16 +318,6 @@ def create(self, vals):
 
     @api.multi
     def write(self, vals):
-        email = vals.get("email")
-        if email:
-            vals["email"] = email.strip()
-            # Push email to non-internal users
-            user_ids = self.mapped("user_ids").filtered("share").ids
-            self.env.cr.execute("""
-                UPDATE res_users
-                SET login=%s
-                WHERE id=ANY(%s)
-            """, [vals["email"], user_ids])
```

Comment:
Do you replace this with a special form where they can change their login ?

Reply:
No, there is a `write` of the login already in _MailChimp_ (that was not there previously). Because of that duplication, modifying the login would fail because of concurrent update in the database.

Reply:
There is no such thing in mailchimp module, or can you point me where it is?

Reply:
You were right, I added the code back and fixed the errors so that it works properly again.